src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -160,6 +166,38 @@ object DepositHandler {
     }
   }
 
+  private def resolveFetchItems(bagitDir: File)(implicit id: String): Try[Unit] = Try {
+    log.debug(s"[$id] Checking fetch.txt")
```

Comment:
My intention was not to check the contents of the fetch.txt before downloading the files.  I added "Checking fetch.txt" to make it clear that the program is checking whether there are files to be fetched.  Only in the case that there is a fetch.txt file, shall the "Resolving files in fetch.txt" text be shown.  I wonder if it is needed to check the contents of the fetch.txt file before downloading the files.